src/data-workspace/data-workspace.js

```diff
@@ -1,9 +1,13 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { useWorkflowContext } from '../workflow-context/index.js'
+import { Navigation } from './navigation.js'
 import { TitleBar } from './title-bar.js'
 
 const DataWorkspace = () => {
     const workflow = useWorkflowContext()
+    const [selectedDataSet, setSelectedDataSet] = useState(
+        workflow.dataSets.length === 1 ? workflow.dataSets[0].id : null
```

Comment:
From the design specs `No data set is selected by default, unless only one is available`